pom.xml

```diff
@@ -43,12 +43,13 @@
     <aspectj.version>1.8.13</aspectj.version>
     <module_name>mod-orders</module_name>
     <http.port>8081</http.port>
+    <jsonschema_paths>acq-models/mod-orders/schemas,acq-models/mod-orders-storage/schemas,raml-util/schemas</jsonschema_paths>
```

Comment:
Looks like `acq-models/common/schemas` is missing here.

Reply:
Just to speed up process I committed the change `acq-models/common/schemas`

![image](https://user-images.githubusercontent.com/43410167/56354491-88f57d80-61dc-11e9-852b-7995ac240339.png)